packages/contracts-bedrock/contracts/L1/L2OutputOracle.sol

```diff
@@ -334,7 +369,7 @@ contract L2OutputOracle is Initializable, Semver {
      * @return Next L2 block number.
      */
     function nextBlockNumber() public view returns (uint256) {
```

Comment:
I lean towards deprecating this function as its semantics change, its obvious if you read the `@notice` and then look at the new implementation

Reply:
Same for `latestBlockNumber`